cloudinary-test-common/src/main/java/com/cloudinary/test/AbstractFoldersApiTest.java

```diff
@@ -37,7 +37,7 @@ public void testRootFolderWithParams() throws Exception {
         String rootFolder2Name = "rootFolderWithParamsTest2" + SUFFIX;
         assertTrue((Boolean) api.createFolder(rootFolder2Name, null).get("success"));
 
-        Thread.sleep(500);
+        Thread.sleep(2000);
```

Comment:
@nitzanj , same here. Relevant for the current change?

Reply:
See above.